tests/www/test_views.py

```diff
@@ -1676,97 +1703,55 @@ def setUp(self):
         self.appbuilder.sm.sync_roles()
         self.add_permission_for_role()
 
-    def login(self, username=None, password=None):
-        role_admin = self.appbuilder.sm.find_role('Admin')
-        tester = self.appbuilder.sm.find_user(username='test')
-        if not tester:
-            self.appbuilder.sm.add_user(
-                username='test',
-                first_name='test',
-                last_name='test',
-                email='test@fab.org',
-                role=role_admin,
-                password='test')
-
-        role_user = self.appbuilder.sm.find_role('User')
-        test_user = self.appbuilder.sm.find_user(username='test_user')
-        if not test_user:
-            self.appbuilder.sm.add_user(
-                username='test_user',
-                first_name='test_user',
-                last_name='test_user',
-                email='test_user@fab.org',
-                role=role_user,
-                password='test_user')
-
-        role_viewer = self.appbuilder.sm.find_role('Viewer')
-        test_viewer = self.appbuilder.sm.find_user(username='test_viewer')
-        if not test_viewer:
-            self.appbuilder.sm.add_user(
-                username='test_viewer',
-                first_name='test_viewer',
-                last_name='test_viewer',
-                email='test_viewer@fab.org',
-                role=role_viewer,
-                password='test_viewer')
-
-        dag_acl_role = self.appbuilder.sm.add_role('dag_acl_tester')
-        dag_tester = self.appbuilder.sm.find_user(username='dag_tester')
-        if not dag_tester:
+    def login(self, username='dag_tester', password='dag_test'):
+        dag_tester_role = self.appbuilder.sm.add_role('dag_acl_tester')
```

Comment:
You have already implemented for `dag_tester`/`dag_acl_tester` in the `login` method of `TestBase`, why it's implemented again here?

Reply:
Ugh, good catch... Off by one when deleting 🤦‍♂️.